scenarios/action_recognition/02_training_hmdb.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>[minor] seems a bit odd if it's a batch to include the number of samples. Don't think fast.ai does it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
its the same things as the `rows` parameter in a way - maybe i should rename to that?